packages/govern-server/src/core/errors.ts

```diff
@@ -0,0 +1,139 @@
+function defineNonEnumerable(instance: object, name: string, value: any) {
```

Comment:
Could get added as ``private`` method to the ``ErrorException`` object cause it isn't used anywhere else and would force us if required to create a ``AbstractError`` class later. 🤷‍♂️